libr/util/idpool.c

```diff
@@ -163,6 +162,70 @@ R_API void* r_id_storage_get(RIDStorage* storage, ut32 id) {
 	return storage->data[id];
 }
 
+R_API bool r_id_storage_get_lowest(RIDStorage *storage, ut32 *id) {
+	r_return_val_if_fail (storage, false);
+	int fd = 0;
+	if (!r_id_storage_get_prev (storage, (ut32 *)&fd) && fd == 0) {
+		return false;
+	}
+	if (!r_id_storage_get_next (storage, (ut32 *)&fd) && fd == 0) {
+		return false;
+	}
+	*id = fd;
+	return true;
+}
+
+R_API bool r_id_storage_get_highest(RIDStorage *storage, ut32 *id) {
+	r_return_val_if_fail (storage, false);
+	int fd = 0;
+	if (!r_id_storage_get_prev (storage, (ut32 *)&fd) && fd == 0) {
+		return false;
+	}
+	*id = fd;
+	return true;
+}
+
+// usually these shall not be used
```

Comment:
remove these comments. We can't provide an API and say "bla bla you should not use it... this is wrong.. bla bla... there is no sorting but we provide prev/next". Either you provide the prev/next API and you expect developers to use it (and also you assume there is some sort of sorting in id-storage) or you don't provide an API.

Reply:
agree